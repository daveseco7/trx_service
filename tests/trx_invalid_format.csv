type,client, tx,amount
deposit,1,1,    1.0
deposit,2, 2,2.0
deposit,1,,2.0
withdrawal,1,4a,1.5
withdrawal,2,5, 3.0