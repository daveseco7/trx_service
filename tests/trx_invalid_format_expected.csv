client,available,held,total,locked
1,1,0.0,1,false
2,2,0.0,2,false